plane/source/docs/quadplane-vtol-tuning-process.rst

```diff
@@ -4,25 +4,85 @@
 Tuning Process Instructions
 ===========================
 
-Setting the aircraft up ready for tuning
-----------------------------------------
+Setting the aircraft up to be ready for tuning
+==============================================
 
 The following parameters should be set correctly based on the specifications of your aircraft.
 
-Battery setting
-^^^^^^^^^^^^^^^
-Parameters used to linearize your motor thrust curve.
+Step 1: Battery and expo settings
+---------------------------------
+
+It is very important to ensure that the thrust curve of your VTOL
+motors is as linear as possible. A linear thrust curve means that
+changes in the actual thrust produced by a motor is directly
+proportional to the thrust being demanded by ArduPilot. If your thrust
+curve is badly non-linear then you will never produce a good tune, and
+in some cases may end up with such a bad tune that your vehicle can
+become completely unstable and crash.
+
+There are 3 common causes of a non-linear thrust curve.
+
+ - voltage sag as throttle is increased or the forward motor is engaged. This is very common in QuadPlanes
+ - incorrect end-point setup in the PWM range you use to your ESCs (see "Motors setup" below)
+ - non-linearity in the thrust produced by your propeller, ESC and motor combination
+
+Start with setting up the voltage range to cope with voltage sag.
+
+- :ref:`Q_M_BAT_IDX <Q_M_BAT_IDX>` : index of the battery to use for voltage measurements on VTOL power system. Zero is the first battery, one for 2nd battery etc
+- :ref:`Q_M_BAT_VOLT_MAX <Q_M_BAT_VOLT_MAX>` : 4.2v x No. Cells for LiPo
+- :ref:`Q_M_BAT_VOLT_MIN <Q_M_BAT_VOLT_MIN>` : 3.3v x No. Cells for LiPo
 
-- :ref:`Q_M_BAT_VOLT_MAX <Q_M_BAT_VOLT_MAX>` : 4.2v x No. Cells
-- :ref:`Q_M_BAT_VOLT_MIN <Q_M_BAT_VOLT_MIN>` : 3.3v x No. Cells
-- :ref:`Q_M_THST_EXPO <Q_M_THST_EXPO>` : 0.55 for 5 inch props, 0.65 for 10 inch props, 0.75 for 20 inch props. This parameter can be derived by thrust stand measurements for optimum results (don’t trust manufacturer data). See :ref:`motor-thrust-scaling` for details
+Note that :ref:`Q_M_BAT_IDX <Q_M_BAT_IDX>` needs to be for the correct
+battery for your VTOL motors. If you have a separate battery for
+forward motors and VTOL motors then make sure you use the right one.
+
+Next setup the thrust expo. If you are setting up a professional
+aircraft then you should invest in a thrust stand so you can
+accurately measure the true thrust for your motor/ESC/propeller
+combination as you vary the throttle. Then you will adjust the expo
```

Comment:
so that you use the region of the thrust curve which is as linear as possible.

so that the thrust between the endpoints is as linear as possible.